assets/js/main.js

```diff
@@ -2,7 +2,8 @@
 /* global jQuery: false */
 /* global cookie: false */
 
-window.GOVUKFrontend.initAll()
+import { initAll } from '../node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js';
```

Comment:
Nitpick: It might be useful to have the link to the bit of the changelog that describes the need for this change included in the commit message.

Reply:
Good idea - I have added that in